proj/cmake/platform_linux.cmake

```diff
@@ -4,6 +4,31 @@ set( CMAKE_VERBOSE_MAKEFILE ON )
 
 set( CINDER_PLATFORM "Posix" )
 
+# Some samples are not available on Linux
+list( APPEND CINDER_SKIP_SAMPLES
+    # Renderer2D is required for:
+    FontSample
+    perlinTest
+    BSpline
+    CairoBasic
+    Wisteria
+    Renderer2dBasic
```

Comment:
Ah, I see. I guess this is good documentation for what left is needed for getting the samples running on linux (or why they never will). As an aside, what is left to get Renderer2D running after this? If you have ideas for that perhaps we should bring to a new github issue.

Reply:
Renderer2D isn't too much of a problem, I've already looked into it a little bit :)